trueline.sh

```diff
@@ -405,6 +422,7 @@ unset TRUELINE_COLORS_DEFAULT
 
 if [[ "${#TRUELINE_SEGMENTS[@]}" -eq 0 ]]; then
     declare -a TRUELINE_SEGMENTS=(
+	'conda_env,black,purple,bold'
```

Comment:
Put this below venv, respect indentation and commented it out (i.e disable it by default).

Reply:
I added the above edits